harness/determined/deploy/aws/templates/vpc.yaml

```diff
@@ -504,7 +521,7 @@ Resources:
   LogGroup:
     Type: AWS::Logs::LogGroup
     Properties:
-      LogGroupName: !Sub /determined/det-${AWS::StackName}
+      LogGroupName: !Sub /${LogGroupPrefix}/${AWS::StackName}
```

Comment:
This seems like a minor but unrelated change / bugfix.

Reply:
it happens to be a fix for https://determined-ai.slack.com/archives/C7E6HPZ5F/p1660787284400649